planb/PBPackageInstaller.m

```diff
@@ -76,49 +75,42 @@ - (NSString *)firstPackageOnImage {
 }
 
 - (void)installApplication {
-  char tmpdir[22];
-  strncpy(tmpdir, "/tmp/planb-pkg.XXXXXX", sizeof tmpdir);
+  NSFileManager *fm = [NSFileManager defaultManager];
 
+  char tmpdir[] = "/tmp/planb-pkg.XXXXXX";
   if (!mkdtemp(tmpdir)) {
     PBLog(@"Error: Could not create temporary installation directory %s.", tmpdir);
     return;
   }
 
-  _mountPoint = [[NSFileManager defaultManager] stringWithFileSystemRepresentation:tmpdir
-                                                                            length:strlen(tmpdir)];
-
-  if ([self runDiskUtilityWithLocation:_packagePath
-                             operation:@"attach"]) {
+  self.mountPoint = [fm stringWithFileSystemRepresentation:tmpdir length:strlen(tmpdir)];
+  if ([self runDiskUtilityWithLocation:self.packagePath operation:@"attach"]) {
     PBLog(@"Mounted %@ (SHA1: %@) to %@",
-          _packagePath,
-          [PBPackageInstaller SHA1ForFileAtPath:_packagePath],
-          _mountPoint);
+          self.packagePath,
+          [self SHA1ForFileAtPath:self.packagePath],
```

Comment:
Is this the only time SHA1ForFileAtPath: is called with self.packagePath? If not, consider caching the hash.